7/apache/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # from https://www.drupal.org/docs/8/system-requirements/drupal-8-php-requirements
-FROM php:7.0-apache
+FROM php:7.1-apache
```

Comment:
Is this supported on Drupal 7?  The URL here is for 8. 😬

Reply:
Ah, https://www.drupal.org/docs/7/system-requirements/drupal-7-php-requirements#php_required is the link you were working from. :+1:

Reply:
(This 7.61 release supposedly is the first with PHP 7.2 support, but it does seem a tad aggressive to jump that far right away. :smile:)